<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestRGBClass</class>
 <widget class="QWidget" name="TestRGBClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TestRGB</string>
  </property>
  <widget class="QLabel" name="video1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>149</y>
     <width>351</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(245, 226, 255);</string>
   </property>
   <property name="text">
    <string>video</string>
   </property>
  </widget>
  <widget class="QLabel" name="video2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>351</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 255, 243);</string>
   </property>
   <property name="text">
    <string>video</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>480</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QComboBox" name="pix1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>YUV420P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RGBA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ARGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BGRA</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="width1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="height1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
   <property name="value">
    <number>300</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="set_fps1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>570</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="height2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>540</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
   <property name="value">
    <number>300</number>
   </property>
  </widget>
  <widget class="QComboBox" name="pix2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>480</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>YUV420P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RGBA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ARGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BGRA</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="open2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>480</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="width2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>510</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="set_fps2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>570</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>300</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QLabel" name="fps1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 7, 50);</string>
   </property>
   <property name="text">
    <string>FPS:</string>
   </property>
  </widget>
  <widget class="QLabel" name="fps2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 10, 83);</string>
   </property>
   <property name="text">
    <string>FPS:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TestRGB.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>open1</sender>
   <signal>clicked()</signal>
   <receiver>TestRGBClass</receiver>
   <slot>Open1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open2</sender>
   <signal>clicked()</signal>
   <receiver>TestRGBClass</receiver>
   <slot>Open2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Open1()</slot>
  <slot>Open2()</slot>
 </slots>
</ui>
